src/components/CreatePostInput/index.js

```diff
@@ -190,9 +177,7 @@ const CreatePostInput = ({
     <>
       <TextInput
         ref={inputRef}
-        onKeyPress={isTopicInDeletedText}
         onChangeText={handlePostTextChanged}
-        // value={message}
         multiline={true}
         numberOfLines={10}
         style={[
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onKeyPress={isTopicInDeletedText}` prop has been removed from the TextInput component. If this event handler is no longer required, this change is fine. However, if you still need to handle key press events, you should add it back.
```diff
+        onKeyPress={isTopicInDeletedText}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->